Beetroot.Lessons/Denys Kovtun/Lesson 7/Homework 7/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Text.RegularExpressions;
+bool Compare(string str1, string str2)
+{
+    if (str1 == str2)
+    {
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+string Analyze(string str)
+{
+    var chars = str.ToCharArray();
+    int digits = 0;
+    int alphabet = 0;
+    int others = 0;
+
+    foreach (char c in chars)
+    {
+        if ((int)Char.GetNumericValue(c) != -1)
+        {
+            digits++;
+        }
+        else if (Regex.IsMatch(c.ToString(), @"[a-z]+$") || Regex.IsMatch(c.ToString(), @"[а-я]+$"))
+        {
+            alphabet++;
+        }
+        else
+        {
+            others++;
+        }
+    }
+    string g = $"Digits:{digits}; Alphabets:{alphabet}; others:{others}";
+    return g;
+}
+string Sort(string str)
+{
+
+    char[] chars = str.ToCharArray();
+    Array.Sort(chars);
+    return new String(chars);
+}
+char[] Duplicate(string stroka)
+{
+    string newStr = "";
```

Comment:
Better to use StringBuilder or char array to not create redundant strings in the heap